### ISSUE TITLE
Bug: Missing meta/main.yml file

### ISSUE THREAD

**Issue by penguineer** [netz39/ble-keykeeper-role#1]
When I try to install this role via requirements (as stated in the Netz39 ansible), I get the following:

```
[WARNING]: - ble-keykeeper-role was NOT installed successfully: this role does not appear to have a meta/main.yml file.
```

Please add this file so that the role can be installed properly.

**Comment by LeSpocky** [comment]
Looks like this was fixed by 61026c56f30e353883dcbef33769afefeb7f06e2, right?

**Comment by penguineer** [comment]
In fact it was. I'm closing this issue.
